lib/assessment.js

```diff
@@ -8,15 +8,20 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(memo, current) {
+  if (current < memo) {
+    memo = current;
```

Comment:
Don't assign to parameters! We do this only in exceptional cases. In general, variables are cheap, so use them!